tuf-spec.md

```diff
@@ -1479,13 +1494,50 @@ it in the next step.
   2. Otherwise, recursively search the list of delegations in
      order of appearance.
 
-    1. If the current delegation is a terminating delegation,
+    1. Abort the update cycle, and report the failure.
```

Comment:
This doesn't look right? That step 1 is "Abort the update cycle"

Reply:
Oops, it looks like I should have removed this with the TAP 3 part.